openpype/hooks/pre_non_python_host_launch.py

```diff
@@ -51,3 +52,17 @@ def execute(self):
 
         if remainders:
             self.launch_context.launch_args.extend(remainders)
+
+        if platform.system().lower() == "windows":
+            # expected behavior - openpype_console opens window with logs
+            # openpype_gui has stdout/stderr available for capturing
+            if "openpype_gui" in os.environ.get("OPENPYPE_EXECUTABLE"):
```

Comment:
Just make sure that it is looked for `openpype_gui` only in filename.
```suggestion
            executable_path = os.environ.get("OPENPYPE_EXECUTABLE")
            executable_filename = ""
            if executable_path:
                executable_filename = os.path.dirname(
                    os.environ.get("OPENPYPE_EXECUTABLE")
                )
            if "openpype_gui" in executable_filename:
```